### PR TITLE
Add Funding

### LINKED ISSUE #51: Add a section for funding
For multiple funding sources, we are required to put their information on our website. Besides that, I think that it might also increase trust in preCICE.
As more and more funding sources are involved, we should add a separate section and adjust the information in the [FAQ](https://www.precice.org/faq/).

I have already collected images on Nextcloud `images/funding/...`

We need mostly images, sometimes a grant number. Links would be nice.

In particular:
* ExaFSA
  * SPPEXA logo
  * `http://www.sppexa.de/`
  * `https://ipvs.informatik.uni-stuttgart.de/SGS/EXAFSA/`
* preDOM
  * DFG logo
  * `http://gepris.dfg.de/gepris/projekt/391150578`
* EuroTech
  * EuroTech logo
  * `http://postdoc.eurotech-universities.eu/`
  * European flag and statement 
> This project has received funding from the European Union’s Horizon 2020 research and innovation programme under the Marie Skłodowska-Curie grant agreement No 754462
* GRS
  * BMWi logo
  * link to project website (e.g. SCCS, or TODO) 

What is needed from Stuttgart? @KyleDavisSA's project?

### PR DESCRIPTION
**Pull request by fsimonis**
This PR adds the funding information to the end of the welcome page.
Closes #51

# Preview
image
_version 2_

### CODE REVIEW

**Review by MakisH — CHANGES_REQUESTED** [review]
From a content structure perspective, I think it makes more sense that this information goes into the About page. There it would also be nice to have the university logos together with the people.

In respect to the aesthetics, maybe a horizontal positioning would be better, at least if it stays in the Welcome page, which is already very bloated at the moment. Again, moving it in the About hides a bit of the bloat.

Small detail regarding the BMWi logo: since our website is in English, I would also use the English version of the logo, see https://www.bmwi.de/Navigation/EN/Home/home.html

**Comment by fsimonis** [comment]
@MakisH I am against putting this information into the about page as it would bloat it up even more.
However, I definitely agree with a horizontal positioning for big screens.

Working on the website again makes me realise that the entire website needs tidying up. Information is all over the place and barely structured. We need to collect user concerns and align the structure towards that.

**Comment by MakisH** [comment]
> @MakisH I am against putting this information into the about page as it would bloat it up even more.
> However, I definitely agree with a horizontal positioning for big screens.

I guess this needs longer discussion, looking at more aspects. For me and at the moment, it looks reasonable that the funding (that supports the developers) is on the same page with the developers. Also, I find it better to have bloated sub-pages, rather than a bloated landing page.

**Comment by uekerman** [comment]
I agree with @MakisH: should go to the about page. But you're right: having a proper discussion on how to arrange things on the web site is sth we should do soonish.

**Comment by fsimonis** [comment]
I moved the list to the about page and made it responsive

**Review by uekerman — APPROVED** [review]

File: 08-about.md [hunk 1]
```diff
@@ -92,6 +94,22 @@ Further contributors include:
 
 The conceptual ideas of preCICE are not completely new, preCICE is an advancement of FSI*ce, which has been developed by Dr. Markus Brenk.
 
+
+## This project is funded by:
```
Comment (MakisH) [inline 08-about.md]:
```suggestion
## Funding

This project is funded by:
```

File: 08-about.md [hunk 2]
```diff
@@ -92,6 +94,22 @@ Further contributors include:
...
+  <li>
+    <a target="_blank" href="http://postdoc.eurotech-universities.eu/"><img src="../assets/funding/eurotech.jpeg" alt="EuroTech"></a>
+  </li>
+  <li>
+    <img src="../assets/funding/eu.png" alt="European Union">
+    <p>This project has received funding from the European Union’s Horizon 2020 research and innovation programme under the Marie Skłodowska-Curie grant agreement No 754462</p>
+  </li>
+</ul>
+
 This website is maintained by
```
Comment (MakisH) [inline 08-about.md:115]:
```suggestion
## Impressum

This website is maintained by
```

File: _sass/precice/_funding.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
+.funding {
+    list-style:none;
+    @include xy-grid;
+    justify-content: center;
+    li {
+        @include xy-cell;
+        @include breakpoint(medium) {
+            @include xy-cell(1 of 3)
+        }
+        margin-top: 1em;
```
Comment (MakisH) [inline _sass/precice/_funding.scss]:
```suggestion
        margin-top: 2em;
```

Comment (MakisH) [reply _sass/precice/_funding.scss]: On my screen the funding logos appear in two rows and with `1em` they look a bit packed.

**Review by MakisH — CHANGES_REQUESTED** [review]
Looks good! Please check my minor suggestions regarding headings and spacing.

File: _sass/precice/_funding.scss (hunk 3 above)
Comment (fsimonis) [reply _sass/precice/_funding.scss]: I agree! `2em` looks better.

**Comment by fsimonis** [comment]
@MakisH with the separate headers it looks a bit strange/redundant:
<details>
<summary>click to expand</summary>

image

</details>

**Comment by MakisH** [comment]
Since we have the heading `## Development`, we definitely need the headings `## Funding` and `## Impressum` as well. Feel free to remove the redundant description, though.

Note that `## This project is funded by:` would be inconsistent (from language and ":" perspective).

**Review by MakisH — APPROVED** [review]

**Pull request merged**